src/platforms/esp32/main/esp32_sys.h

```diff
@@ -32,5 +32,8 @@ extern void *event_descriptors[EVENT_DESCRIPTORS_COUNT];
 int open_event_descriptor(void *ptr);
 void close_event_descriptor(int index);
 int find_event_descriptor(void *ptr);
+void *get_event_ptr(int i);
```

Comment:
Just as a point of information, I added get_event_ptr in order to allow us to pass an event descriptor back to the Erlang side on a call to gen_server:accept, which in turn is used to initialize the accepting socket.  The event descriptor then needs to be used to look up the netconn ptr.